models/contributor.py

```diff
@@ -90,6 +90,27 @@ async def generate_image(self) -> Image:
 
         draw = ImageDraw.Draw(image)
 
+        def contributor_of_the() -> str:
```

Comment:
What if the date is not equal to them .. show "contributor of the " only?
Add a default as well with "last n days."

Reply:
@diwash007 Done!